### PR TITLE
Transition spreadsheet

### PR DESCRIPTION
**Pull request by sanjanasrivastava**

### CODE REVIEW

File: tests/tm_csvs/make_lamb_stew.csv [hunk 1]
```diff
@@ -1,56 +0,0 @@
-STEP 1,Unnamed: 1
-input-objects,
-lamb.n.01_1,
-output-objects,
-diced__lamb.n.01_1,
```
Comment (ChengshuLi) [inline tests/tm_csvs/make_lamb_stew.csv:5]: why do we remove these csv files? Cuz the task no longer exist?

**Review by ChengshuLi — APPROVED** [review]
LGTM with some minor comments!

File: bddl/activity_definitions/make_brownies/problem0.bddl [hunk 2]
```diff
@@ -43,13 +44,21 @@
...
+        (inside walnut.n.01_6 bowl.n.01_2) 
+        (inside walnut.n.01_7 bowl.n.01_2) 
+        (ontop bowl.n.01_2 countertop.n.01_2)
+        (ontop electric_mixer.n.01_1 countertop.n.01_2) 
         (ontop mixing_bowl.n.01_1 countertop.n.01_2) 
         (inside bowl.n.01_1 electric_refrigerator.n.01_1) 
         (inside tupperware.n.01_1 electric_refrigerator.n.01_1) 
         (inroom electric_refrigerator.n.01_1 kitchen) 
         (inroom oven.n.01_1 kitchen) 
+        (inroom microwave.n.02_1 kitchen)
```
[the remark this answers is not in the document]
Comment (sanjanasrivastava) [reply bddl/activity_definitions/make_brownies/problem0.bddl:61]: The butter is melted (with any heatSource), the microwave was added for convenience for that stage

File: bddl/activity_definitions/make_homemade_bird_food/problem0.bddl [hunk 3]
```diff
@@ -43,14 +43,19 @@
...
     
     (:goal 
         (and 
-            (real ?bird_feed.n.01_1)
+            (real ?diced__apple.n.01_1)
+            (covered ?pinecone.n.01_1 ?peanut_butter.n.01_1)
+            (covered ?pinecone.n.01_1 ?almond.n.02_1)
+            (covered ?pinecone.n.01_1 ?diced__apple.n.01_1)
+            (covered ?pinecone.n.01_1 ?sunflower_seed.n.01_1)
+            (covered ?pinecone.n.01_1 ?raisin.n.01_1)
```
[the remark this answers is not in the document]
Comment (sanjanasrivastava) [reply bddl/activity_definitions/make_homemade_bird_food/problem0.bddl:58]: Yeah :( we could just change it to mixing those things together

File: tests/tm_csvs/make_lamb_stew.csv (hunk 1 above)
Comment (sanjanasrivastava) [reply tests/tm_csvs/make_lamb_stew.csv:5]: Either it doesn't exist or it's no longer a TM recipe

**Pull request merged**